### PR TITLE
DOC: Command fixes for release branch merges

### PR DESCRIPTION
**Pull request by thewtex**
`git pull` should not have a slash between the remote and branch.

`git push` for the `release` branch update had the wrong branch.

### CODE REVIEW

**Review by hjmjohnson — APPROVED** [review]

File: Documentation/docs/contributing/index.md [hunk 1]
```diff
@@ -339,16 +339,16 @@ then:
 
 ```bash
 git checkout release
-git pull --ff-only upstream/release
+git pull --ff-only upstream release
 git merge --no-ff release-X.X
-git push upstream release-X.X
+git push upstream release
```
Comment (dzenanz) [inline Documentation/docs/contributing/index.md:344]: Was this change intentional?

Comment (thewtex) [reply Documentation/docs/contributing/index.md:344]: Yes, as noted in the commit message.
